src/operator/contrib/count_sketch.cu

```diff
@@ -129,8 +129,7 @@ inline void CountSketchForward(const Tensor<gpu, 2, DType> &out,
                                     nthreads, out_ptr+bstart*out_dim, h_ptr,
                                     s_ptr, in_ptr+bstart*in_dim, batchlen,
                                     in_dim, out_dim);
-    MSHADOW_CUDA_POST_KERNEL_CHECK(sketch_forward_kernel);
-    // cudaThreadSynchronize();
+    CHECK_EQ(cudaDeviceSynchronize(), cudaSuccess) << "Problem occured!";
```

Comment:
Consider adding more info to the log (e.g. cudaSuccess value).

Reply:
ok

Reply:
Done.